File: src/gradle-tooling-extension/java/com/demonwav/mcdev/platform/mcp/gradle/tooling/McpModel.java
/*
 * Minecraft Dev for IntelliJ
 *
 * https://minecraftdev.org
 *
 * Copyright (c) 2018 minecraft-dev
 *
 * MIT License
 */

package com.demonwav.mcdev.platform.mcp.gradle.tooling;

import java.util.Set;

public interface McpModel {
    String getMinecraftVersion();
    String getMcpVersion();
    Set<String> getMappingFiles();
}

File: src/main/kotlin/com/demonwav/mcdev/platform/mcp/gradle/McpModel.kt
/*
 * Minecraft Dev for IntelliJ
 *
 * https://minecraftdev.org
 *
 * Copyright (c) 2017 minecraft-dev
 *
 * MIT License
 */

package com.demonwav.mcdev.platform.mcp.gradle

interface McpModel {

    val minecraftVersion: String
    val mcpVersion: String
    val mappingFiles: Set<String>
}
